pipelinewise/fastsync/commons/tap_mongodb.py

```diff
@@ -153,6 +154,54 @@ def transform_value(value: Any, path) -> Any:
     return value
 
 
+def get_connection_string(config: Dict):
+    """
+    Generates a MongoClientConnectionString based on configuration
+    Args:
+        config: DB config
+
+    Returns: A MongoClient connection string
+    """
+    srv = config.get('srv') == 'true'
+
+    # Default SSL verify mode to true, give option to disable
+    verify_mode = config.get('verify_mode', 'true') == 'true'
+    use_ssl = config.get('ssl') == 'true'
+
+    connection_query = {
+        'readPreference': 'secondaryPreferred',
+        'authSource': config['auth_database'],
+    }
+
+    if config.get('replica_set'):
+        connection_query['replicaSet'] = config['replica_set']
+
+    if use_ssl:
+        connection_query['tls'] = 'true'
+
+    # NB: "sslAllowInvalidCertificates" must ONLY be supplied if `SSL` is true.
+    if not verify_mode and use_ssl:
+        connection_query['tlsAllowInvalidCertificates'] = 'true'
+
+    # looks like mongo dump and pymongo differ in ssl params
```

Comment:
We can use `ssl`, mongodump expects `ssl` and pymongo supports both `tls` and `ssl` (`ssl` is alias for `tls`).